src/Razor/src/Microsoft.VisualStudio.LanguageServices.Razor/DynamicFiles/RazorDynamicFileInfoProvider.cs

```diff
@@ -489,9 +489,11 @@ public void SetSupportsDiagnostics(bool value)
 
         public IRazorDocumentExcerptServiceImplementation? GetExcerptService() => _documentExcerptService;
 
-        public IRazorSpanMappingService? GetMappingService() => _spanMappingService;
+        public IRazorSpanMappingService? GetSpanMappingService() => _spanMappingService;
 
         public TextLoader GetTextLoader(string filePath) => _textLoader;
+
+        public IRazorMappingService? GetMappingService() => null;
```

Comment:
As per the comment on `DefaultDynamicDocumentContainer`, I'm not sure I understand why this is okay to be null.